src/Entity/Register.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\commerce_pos\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBase;
+
+/**
+ * Defines the Register entity.
+ *
+ * @ConfigEntityType(
+ *   id = "register",
+ *   label = @Translation("Register"),
+ *   handlers = {
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "list_builder" = "Drupal\commerce_pos\RegisterListBuilder",
+ *     "form" = {
+ *       "add" = "Drupal\commerce_pos\Form\RegisterForm",
+ *       "edit" = "Drupal\commerce_pos\Form\RegisterForm",
+ *       "delete" = "Drupal\commerce_pos\Form\RegisterDeleteForm"
+ *     },
+ *     "route_provider" = {
+ *       "html" = "Drupal\commerce_pos\RegisterHtmlRouteProvider",
+ *     },
+ *   },
+ *   config_prefix = "register",
+ *   admin_permission = "administer site configuration",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "label",
+ *     "uuid" = "uuid"
+ *   },
+ *   links = {
+ *     "canonical" = "/admin/structure/pos/register/register/{register}",
```

Comment:
1) Why aren't the urls under /admin/commerce/? Either /admin/commerce/pos or /admin/commerce/config/pos?
2) Why do registers have a canonical url, they're not meant to be viewed?